### PR TITLE
User messages

### PR DESCRIPTION
**Pull request by TobiasKadelka**
Solves inconsistencies and ambiguities in messages displayed to the user.

Close https://github.com/psyinfra/onyo/issues/423
Close https://github.com/psyinfra/onyo/issues/430
Close https://github.com/psyinfra/onyo/issues/442

### CODE REVIEW

File: onyo/lib/commands.py [hunk 1]
```diff
@@ -437,7 +437,10 @@ def onyo_mv(inventory: Inventory,
             inventory.rename_directory(sources[0], destination)
 
     if inventory.operations_pending():
-        ui.print('The following will be moved:')
+        if subject == "mv":
+            ui.print('The following will be moved:')
+        else:
+            ui.print('The following will be renamed:')
```
Comment (bpoldrack) [inline onyo/lib/commands.py:443]: Minor suggestion, but for such minimal thing I'd prefer a one-liner.
```suggestion
        ui.print("The following will be {}.".format("moved" if subject == "mv" else "renamed"))
```

**Review by bpoldrack — APPROVED** [review]
Thanks, @TobiasKadelka!

One minor suggestion, but not a blocker.

**Comment by TobiasKadelka** [comment]
Thanks for the suggestion, I updated the PR accordingly 👍

**Comment by bpoldrack** [comment]
Thanks, @TobiasKadelka!

**Pull request merged**